manifests/manifests_test.go

```diff
@@ -0,0 +1,47 @@
+package manifests_test
+
+import (
+	"bytes"
+	"fmt"
+	"maistra.io/api/manifests"
```

Comment:
I don't think you need this

Reply:
The package of the test is `manifests_test` - so I need it.

Reply:
why? just rename it to manifests

Reply:
I think the convention is to "hide" tests in the dedicated `_test` package so that consumer lib/project won't have them leaked.

Reply:
not in istio at least

Reply:
And about leaking - it seems that `_test.go` suffix is enough, so my argument was invalid.

Reply:
042bca4